site/content/en/docs/tasks/troubleshooting/troubleshooting_jobs.md

```diff
@@ -60,6 +60,45 @@ Job is called `my-job` in the `my-namespace` namespace.
    job-my-job-19797   user-queue   cluster-queue   9m45s
    ```
 
+## Does my cluster-queue match the job resource requests?
+
+When you submit a job that has a resource request, for example:
+
+```bash
+$ kubectl get jobs job-0-9-size-6 -o json | jq -r .spec.template.spec.containers[0].resources
+```
+```console
+{
+  "limits": {
+    "cpu": "2"
+  },
+  "requests": {
+    "cpu": "2"
+  }
+}
+```
+
+If your cluster queue does not have a definition for the _requests_ the job cannot be admitted. For the job above you would want to ensure that you've defined "cpu" under `resourceGroups`, as shown below:
+
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "cluster-queue"
+spec:
+  namespaceSelector: {}
+  resourceGroups:
+  - coveredResources: ["cpu"]
+    flavors:
+    - name: "default-flavor"
+      resources:
+      - name: "cpu"
+        nominalQuota: 40
+```
+
+Generally speaking, if you find that workloads are not admitted, a good debugging strategy is to check that you've covered resources that are requested, and then carefully describe each of the job and workload, along with looking at logs for kueue. You can typically find a log or condition message that gives a hint to why the queue is not moving.
+
```

Comment:
this is already explained in the other sections.

Reply:
I am going to say here that redundancy is OK, especially in documentation. I don't think we want to be maintaining equivalent information (e.g., some spec or definition) in two places, but reminders about "here is a good way to debug" are always good to have multiple times. The user reader very likely has not seen/read all the "the other sections."

Reply:
I did wind up removing it, resolving!